.github/CODEOWNERS

```diff
@@ -39,3 +39,6 @@ translate.yaml                                          @DataDog/corpweb
 .translate                                              @DataDog/corpweb
 docker-compose-docs.yml                                 @DataDog/corpweb
 .htmltest.yml                                           @DataDog/corpweb
+
+# Cloud Cost Management
+documentation/content/*/cloud_cost_management/_index.md         @DataDog/cloud-cost-management-visibility
```

Comment:
I would suggest making this en only, as otherwise you'll get pinged for every translation build